### PR TITLE
Tl edits

### PR DESCRIPTION
**Pull request by laderast**
I have overwritten the previous image files with shots of the new interface.  I also added a flow diagram to orient users to the actual process of submitting jobs with PROOF.

I need someone to check:

- "Cromwell Server" vs "PROOF Server". In the new interface, it's referred to as "PROOF Server". I've updated the terminology to reflect that, **except in the Cromwell Server** section. I added a note at the end of this section noting that it's going to be referred to as "PROOF Server" in the instructions.
- Also, almost none of the Tracking interface was changed so I did not update those screenshots.

I think I caught most everything else.

### CODE REVIEW

**Comment by tefirman** [comment]
New screenshots look awesome, @laderast, thanks for updating those!!! I fixed a couple of small typos and removed the screenshot advertising the GitHub link, sounded like we wanted to limit outside contributions for the first few weeks.

File: _datascience/assets/proof_101_shiny_app_code.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_code.png:1]: you may want to blackout your user name

Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_code.png:1]: you many want to block out your username

File: _datascience/assets/proof_101_shiny_app_help.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_help.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_help2.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_help2.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_track_jobs_1.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_track_jobs_1.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_track_jobs_2.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_track_jobs_2.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shinyapp_logged_in.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shinyapp_logged_in.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shinyapp_start_server_2.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shinyapp_start_server_2.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shinyapp_start_server_4.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shinyapp_start_server_4.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shinyapp_validate.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shinyapp_validate.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_track_jobs_1.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_track_jobs_1.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_track_jobs_2.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_track_jobs_2.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_code.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_code.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_help.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_help.png:1]: you may want to block out your username

File: _datascience/assets/proof_101_shiny_app_help2.png
Comment (sitapriyamoorthi) [inline _datascience/assets/proof_101_shiny_app_help2.png:1]: you may want to block out your username

File: _datascience/proof.md
Comment (sitapriyamoorthi) [inline _datascience/proof.md:1]: lets try to keep PROOF all caps consistent across the doc

**Review by sitapriyamoorthi — CHANGES_REQUESTED** [review]
Left some minor comments

**Pull request merged**